frontend/pages/playscreen.vue

```diff
@@ -85,33 +85,35 @@
     </div>
 
     <!-- Spielende-Anzeige -->
-    <div class="game-over" v-if="isGameOver && isTrisightMode">
-      <div class="final-score">Dein Score: {{ score }}</div>
-      <!-- Neue Buttons für Home und Neue Runde -->
-      <div class="button-group">
-        <NuxtLink to="/" class="button">Home</NuxtLink>
-        <button @click="startNewRound" class="button">Neue Runde</button>
-      </div>
-    </div>
+    <!-- Verwende das ResultsOverlay-Overlay -->
```

Comment:
I dont know why but when I execute any test configs of my saved ones i dont get the ResultOverlay aftwerwards, the screen is just blank. Is that still in the working process or is there an error occuring? It works well for the Trisight Mode and i really like how it looks. Same with the profile view, very good job!!!

Reply:
should be fixed